### ISSUE TITLE
beamerarticle doesn't work because of  \setbeamercovered undefined

### ISSUE THREAD

**Issue by dbitouze** [josephwright/beamer#365]
The following MCE:
```latex
\documentclass{article}
\usepackage{beamerarticle}
%
\begin{document}
\begin{frame}
  Foo.
\end{frame}
\end{document}
```
cannot compile because of the error `! Package etoolbox Error: \setbeamercovered undefined.`

**Comment by louisstuart96** [comment]
Seems like it's fixed in [59118a9](https://github.com/josephwright/beamer/commit/59118a96072c8c120ab4937bb7c63111dc111b85).

**Comment by josephwright** [comment]
Indeed: should be OK in the release and on the master branch ...
